### PR TITLE
Remove SemVer from Slick module

### PR DESCRIPTION
**Pull request by mdedetrich**
See discussion at https://github.com/apache/incubator-pekko-projection/issues/110 for why

### CODE REVIEW

**Comment by pjfanning** [comment]
I'm not sure why #110 means we don't have a stable set of APIs.

**Comment by mdedetrich** [comment]
> I'm not sure why #110 means we don't have a stable set of APIs.

Updating slick will break binary compatibility (most likely)

**Comment by mdedetrich** [comment]
Actually I realized that technically speaking we only need to make the jdbc module of this project not semver compatible, let me update PR

**Review by pjfanning — APPROVED** [review]
lgtm

File: build.sbt [hunk 1]
```diff
@@ -82,7 +82,9 @@ lazy val slick =
     .settings(Defaults.itSettings)
     .settings(Dependencies.slick)
     .settings(AutomaticModuleName.settings("pekko.projection.slick"))
-    .settings(name := "pekko-projection-slick")
+    .settings(
+      name := "pekko-projection-slick",
+      versionScheme := None)
```
Comment (He-Pin) [inline build.sbt:87]: Good, but better with some common

**Review by He-Pin — APPROVED** [review]
Lgtm

File: build.sbt (hunk 1 above)
Comment (mdedetrich) [reply build.sbt:87]: What do you mean?

**Pull request merged**